projects/observability/src/pages/apis/service-detail/instrumentation/components/total-score.component.ts

```diff
@@ -0,0 +1,74 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { ServiceInstrumentationService } from '../service-instrumentation.service';
+
+@Component({
+  styleUrls: ['./total-score.component.scss'],
+  selector: 'ht-service-instrumentation-total-score',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="service-instrumentation-total-score">
+      <ht-progress-circle
+        [percent]="serviceScore"
+        [colorLight]="scoreColors?.light"
+        [colorDark]="scoreColors?.dark"
+      ></ht-progress-circle>
+
+      <div>
+        <h4 class="heading">{{ this.getScoreLabel() }}</h4>
+
+        <p class="description">
+          {{ this.getDescriptionForScore() }}
+        </p>
+
+        <div class="legend">
+          <div class="dot">
+            <span [style.background-color]="'#dc3d43'"></span>
+            <label>0-49</label>
+          </div>
+
+          <div class="dot">
+            <span [style.background-color]="'#ffa01c'"></span>
+            <label>50-69</label>
+          </div>
+
+          <div class="dot">
+            <span [style.background-color]="'#3d9a50'"></span>
+            <label>70-100</label>
+          </div>
```

Comment:
nit: See a bit of redundancy, can try maintaining an array with `label` and `color` and iterate over it here.

Reply:
✅ done